[2020-06-03 08:00:49,803] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:00:49,814] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:00:49,814] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:00:49,814] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-03 08:00:49,814] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:00:49,830] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): covid_data> on 2020-05-31T00:00:00+00:00
[2020-06-03 08:00:49,833] {standard_task_runner.py:53} INFO - Started process 28162 to run task
[2020-06-03 08:00:49,899] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [running]> nineleaps
[2020-06-03 08:00:50,665] {logging_mixin.py:112} INFO - csv loaded successfully.
[2020-06-03 08:00:50,666] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-03 08:00:50,669] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=covid_dag, task_id=covid_data, execution_date=20200531T000000, start_date=20200603T023049, end_date=20200603T023050
[2020-06-03 08:00:59,796] {logging_mixin.py:112} INFO - [2020-06-03 08:00:59,795] {local_task_job.py:103} INFO - Task exited with return code 0
[2020-06-03 08:16:40,170] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:16:40,176] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:16:40,176] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:16:40,176] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-03 08:16:40,176] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:16:40,193] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): covid_data> on 2020-05-31T00:00:00+00:00
[2020-06-03 08:16:40,195] {standard_task_runner.py:53} INFO - Started process 30263 to run task
[2020-06-03 08:16:40,250] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [running]> nineleaps
[2020-06-03 08:16:40,709] {taskinstance.py:1128} ERROR - string indices must be integers
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nineleaps/airflow/dags/covid_data.py", line 69, in get_covid_data
    csv_writer.writerow([context['execution_date'], i['State UT'], i[str(context['execution_date']).split('T')[0]]])
TypeError: string indices must be integers
[2020-06-03 08:16:40,711] {taskinstance.py:1151} INFO - Marking task as UP_FOR_RETRY
[2020-06-03 08:16:50,167] {logging_mixin.py:112} INFO - [2020-06-03 08:16:50,166] {local_task_job.py:103} INFO - Task exited with return code 1
[2020-06-03 08:22:56,550] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:22:56,557] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:22:56,557] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:22:56,558] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-03 08:22:56,558] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:22:56,573] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): covid_data> on 2020-05-31T00:00:00+00:00
[2020-06-03 08:22:56,576] {standard_task_runner.py:53} INFO - Started process 31225 to run task
[2020-06-03 08:22:56,628] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [running]> nineleaps
[2020-06-03 08:22:57,067] {taskinstance.py:1128} ERROR - string indices must be integers
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/airflow/models/taskinstance.py", line 966, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 113, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python2.7/dist-packages/airflow/operators/python_operator.py", line 118, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/nineleaps/airflow/dags/covid_data.py", line 68, in get_covid_data
    csv_writer.writerow([context['execution_date'], i['State UT'], i[str(context['execution_date']).split('T')[0]]])
TypeError: string indices must be integers
[2020-06-03 08:22:57,071] {taskinstance.py:1151} INFO - Marking task as UP_FOR_RETRY
[2020-06-03 08:23:06,543] {logging_mixin.py:112} INFO - [2020-06-03 08:23:06,542] {local_task_job.py:103} INFO - Task exited with return code 1
[2020-06-03 08:24:22,513] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:24:22,522] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:24:22,522] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:24:22,522] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-03 08:24:22,523] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:24:22,539] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): covid_data> on 2020-05-31T00:00:00+00:00
[2020-06-03 08:24:22,545] {standard_task_runner.py:53} INFO - Started process 31408 to run task
[2020-06-03 08:24:22,611] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [running]> nineleaps
[2020-06-03 08:24:26,558] {logging_mixin.py:112} INFO - csv loaded successfully.
[2020-06-03 08:24:26,558] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-03 08:24:26,562] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=covid_dag, task_id=covid_data, execution_date=20200531T000000, start_date=20200603T025422, end_date=20200603T025426
[2020-06-03 08:24:32,507] {logging_mixin.py:112} INFO - [2020-06-03 08:24:32,506] {local_task_job.py:103} INFO - Task exited with return code 0
[2020-06-03 08:29:00,920] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:29:00,928] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [queued]>
[2020-06-03 08:29:00,929] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:29:00,929] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-03 08:29:00,929] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-03 08:29:00,945] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): covid_data> on 2020-05-31T00:00:00+00:00
[2020-06-03 08:29:00,948] {standard_task_runner.py:53} INFO - Started process 32014 to run task
[2020-06-03 08:29:01,017] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_dag.covid_data 2020-05-31T00:00:00+00:00 [running]> nineleaps
[2020-06-03 08:29:05,298] {logging_mixin.py:112} INFO - csv loaded successfully.
[2020-06-03 08:29:05,299] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-03 08:29:05,310] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=covid_dag, task_id=covid_data, execution_date=20200531T000000, start_date=20200603T025900, end_date=20200603T025905
[2020-06-03 08:29:10,918] {logging_mixin.py:112} INFO - [2020-06-03 08:29:10,916] {local_task_job.py:103} INFO - Task exited with return code 0
